src/pages/student/students/StudentDetail/index.js

```diff
@@ -66,6 +71,30 @@ const StudentDetail = () => {
     }
   };
 
+  const iconDisplay = (activityDetail) => {
+    if (activityDetail === 'App\\Models\\Quiz') {
```

Comment:
Need ba ing ana nga string for identifying activity detail?

Reply:
I think mao ray way Sir, na ma identify na Quizzes related or Follow/Unfollow ang kana na activity.